lib/main.coffee

```diff
@@ -0,0 +1,60 @@
+{CompositeDisposable} = require 'atom'
+{parseString} = require 'xml2js'
+path = require 'path'
+
+module.exports =
+  config:
+    executablePath:
+      title: 'CSSLint Executable Path'
+      type: 'string'
+      # default: path.join(__dirname, '..', 'node_modules', '.bin', 'csslint.cmd')
```

Comment:
How would I include it? As far as I could tell it doesn't provide the "cli wrapper mode" that things like jshint do.